enterprise/reporting/reporting.go

```diff
@@ -276,6 +313,7 @@ func (r *DefaultReporter) getReports(currentMs, aggregationIntervalMin int64, sy
 func (r *DefaultReporter) getAggregatedReports(reports []*types.ReportByStatus) []*types.Metric {
 	metricsByGroup := map[string]*types.Metric{}
 	maxReportsCountInARequest := r.maxReportsCountInARequest.Load()
+	bucket, _ := r.getAggregationBucketMinute(reports[0].ReportedAt, 1)
```

Comment:
`bucket` is only for collecting less sample_events right ? Let's rename it explicitly to `sample_event_bucket` ?

```suggestion
	bucket, _ := r.getAggregationBucketMinute(reports[0].ReportedAt, int64(r.eventSamplingDuration.Load().Minutes()))
```